iscsi/targetd/provisioner/iscsi-provisioner.go

```diff
@@ -108,6 +108,40 @@ func (p *iscsiProvisioner) getAccessModes() []v1.PersistentVolumeAccessMode {
 	}
 }
 
+func createPVSource(options controller.VolumeOptions, portals []string, lun int32) v1.PersistentVolumeSource {
+	if options.PVC.Annotations["os"] == "windows" {
+		return v1.PersistentVolumeSource{
+			FlexVolume: &v1.FlexPersistentVolumeSource{
+				Driver: "microsoft.com/iscsi.cmd",
```

Comment:
I'm slightly wary of putting a dependency on a vendor-specific driver in this package that is supposed to be vendor-neutral.  Would it be better to put this under the same repo where the flex driver is located at?

Reply:
yes, feel free to fork. The targetd provisioner doesn't have an active maintainer so if you want to repurpose it to support also iscsi on windows workers, & have free reign to make changes to it, we can link to the fork from here.

Reply:
Sounds good. @KnicKnic will it make sense to host the Linux targetd provisioner fork (with the above PR) in https://github.com/microsoft/ ?

Reply:
Instead of forking, how about make the flexvol plugin `Driver` field configurable from the storage class? Simpler.

Reply:
> Sounds good. @KnicKnic will it make sense to host the Linux targetd provisioner fork (with the above PR) in https://github.com/microsoft/ ?

@ddebroy - I work with @KnicKnic. That sounds good; we can host it at microsoft/k8s-storage-plugins

Reply:
@anusha-ragunathan the Driver field may be made configurable but the [options](https://github.com/kubernetes-incubator/external-storage/pull/1226/files#diff-d5f705cb57d27aa7dee80b63d3cb25efR117) keys are pretty tightly coupled with the implementation of the flexvolume driver.